webpack.prod.js

```diff
@@ -0,0 +1,12 @@
+const merge = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+   devtool: 'source-map',
+   plugins: [
+     new UglifyJSPlugin({
+       sourceMap: true
+     })
+   ]
+ });
```

Comment:
In this production configuration you can also set the `sass` compiler to minify.